features/file_sourcing.feature

```diff
@@ -0,0 +1,62 @@
+Feature: Sourcing tcl files
+
+  Scenario: Display stack trace when there is an error in a sourced file
+    Given a file named "features/test.feature" with:
+      """
+      Feature:
+        Scenario:
+          Given testing file sourcing
+      """
+    And a file named "features/support/env.rb" with:
+      """
+      require 'cucumber/tcl'
+      """
+    And a file named "features/support/env.tcl" with:
```

Comment:
> I'll modify the name of the scenario as suggested.

Do you mean modify the name of the *file*?

Yeah, let's add two new tickets: one for loading `env.tcl` first and then another for supporting `--dry-run` mode.